Resources/Storage/StorageElement.py

```diff
@@ -131,7 +131,7 @@ def __init__( self, name, plugins = None, vo = None ):
         return
       self.vo = result['Value']
     self.opHelper = Operations( vo = self.vo )
-    
+
```

Comment:
Someone amongst developers has a setting that sets plans spaces and doesn't trim the lines' end before saving... Sorry, this is always creating a lot of "beautifying changes" and it seems these spaces reappear regularly...